java/arcs/core/data/SchemaFields.kt

```diff
@@ -23,7 +23,10 @@ sealed class FieldType(
     }
 
     /** A reference to an entity. */
-    data class EntityRef(val schemaHash: String) : FieldType(Tag.EntityRef) {
+    data class EntityRef(
```

Comment:
maybe better to put the annotations list in `FieldType` directly? the manifest supports annotations on any field, so it's more consistent.

Reply:
done